subtrack.MAUI/Shared/SubscriptionItem.razor

```diff
@@ -0,0 +1,46 @@
+﻿<div class="card border-primary mb-3" style="max-width: 20rem;">
+    <div class="card-body">
+        <div class="d-flex justify-content-between fs-5">
+            <div>@Subscription.Name</div>
+            <div class="text-right">@($"{Subscription.Cost:C}")</div>
+        </div>
+    </div>
+    <div class="card-body">
+        <div class="d-flex justify-content-between fs-6">
+            <div>@(Subscription.LastPayment.Day + GetDateSuffix(Subscription.LastPayment.Day))</div>
+            <div class="text-right">@(Subscription.IsAutoPaid ? "Auto" : string.Empty)</div>
+        </div>
+    </div>
+</div>
+
+
+
+@code {
+    [Parameter]
+    [EditorRequired]
+    public Subscription Subscription { get; set; }
+
+    public string GetDateSuffix(int day)
```

Comment:
This solution is imo more readable: https://stackoverflow.com/questions/2050805/getting-day-suffix-when-using-datetime-tostring

Not a requirement but this might be a nice use case of pattern matching: 
https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/functional/pattern-matching

Reply:
That was a nice suggestion! Changed the method to this :
```cs
    public string GetDateSuffix(int day)
    {
        return day switch
        {
            1 or 21 or 31 => "st",
            2 or 22 => "nd",
            3 or 23 => "rd",
            _ => "th"
        };
    }
```